CLAUDE.md

```diff
@@ -0,0 +1,25 @@
+# WordPress Studio Development Guide
```

Comment:
We should probably remove that file, I'm not sure we should be adding the config files/intructions for all the AIs at the moment.